components/Common/ShellBox/index.tsx

```diff
@@ -0,0 +1,43 @@
+import React, { useCallback } from 'react';
+import { FormattedMessage } from 'react-intl';
+import { useCopyToClipboard } from '../../../hooks/useCopyToClipboard';
+import styles from './index.module.scss';
+import ReactDomServer from 'react-dom/server';
+
+interface Props {
+  children: React.ReactNode | undefined;
```

Comment:
```suggestion
```
I think there is no need to pass `children` type explicitly when using `React.PropsWithChildren<Props>`